app/src/main/java/com/github/se/polyfit/MainActivity.kt

```diff
@@ -75,7 +75,8 @@ class MainActivity : ComponentActivity() {
                 })
           }
           composable(Route.OverviewScan) {
-            IngredientsOverview(navigation::goBack, {}, {}, emptyList())
+            IngredientsOverview(
+                navigation::goBack, navigation::navigateToRecipeRecommendationFlow, {}, emptyList())
```

Comment:
what's the purpose of the empty fun {}

Reply:
We don't know how to integrate to the next part.